### PR TITLE
fix send emoji

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes RNBW-4553
Figma link (if any):

## What changed (plus any additional context for devs)
removed feature flag and fixed the logic

## Screen recordings / screenshots

https://cloud.skylarbarrera.com/Screen-Recording-2022-09-29-14-27-08.mp4

^ ignore the double of addresses in the list, I used that for testing

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Comment by estebanmino** [comment]
not coming from this PR but found another instance of this happening, while adding contact from send sheet https://www.loom.com/share/5f0407241e27490c8bf25cd8dd858217

**Comment by skylarbarrera** [comment]
updated PoW: https://cloud.skylarbarrera.com/Screen-Recording-2022-09-30-12-32-27.mp4

File: src/components/contacts/SwipeableContactRow.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ const SwipeableContactRow = (
   forwardedRef
 ) => {
   const swipeableRef = useRef();
+  console.log(color);
```
Comment (estebanmino) [inline src/components/contacts/SwipeableContactRow.js:70]: uh

**Review by estebanmino — APPROVED** [review]

**Review by derHowie — APPROVED** [review]
lgtm

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
Still seeing the issue @estebanmino reported above about the "add contact" flow not updating

https://user-images.githubusercontent.com/41752922/193649828-67861a2a-7ec1-4e5d-8819-e0c11a6b3b93.mov

Also, if I change from an emoji to an image in my library, the send sheet doesn't update

https://user-images.githubusercontent.com/41752922/193650099-8903b2f8-0dc8-4ae1-955f-9961b0c7d081.mov

**Comment by skylarbarrera** [comment]
@ibrahimtaveras00 these are both expected behavior unfortunately, we do not check imported/watched wallets here which is causing this behavior 

checked a build from earlier this year and same thing

**Review by ibrahimtaveras00 — APPROVED** [review]
in light of above comment, this is good now 👍🏽

**Pull request merged**